app/javascript/components/explore/ExploreDisplayTabs.js

```diff
@@ -356,9 +358,18 @@ export default function ExploreDisplayTabs({
         <div className={showViewOptionsControls ? 'col-md-2 ' : 'hidden'}>
           <div className="view-options-toggle">
             <FontAwesomeIcon className="fa-lg" icon={faCog}/> OPTIONS
-            <button className="action" onClick={toggleViewOptions}>
+            <button className="action"
+              onClick={toggleViewOptions}
+              title="hide options"
+              data-analytics-name="explore-view-options-hide">
```

Comment:
Let's omit page-name-like information from `data-analytics-name`, which would help compare usage of reusable components (like a toggler) in different contexts.

```suggestion
              data-analytics-name="options-hide">
```

Alternatively, adding e.g. `className="options-hide"` would enable similar comparisons, while keeping `data-analytics-name` globally unique.